pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,65 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitment
+author: Mixxx, April
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+# &#35;BlackLivesMatter
+
+Alongside the protesters in the US and around the world, the Mixxx team unequivocally condemns police brutality.
+The systemic racism that burdens people of color every day is the responsibility of white people around the world to address.
+Mixxx' core development team feels it necessary not to remain silent on this issue, for ["silence in the face of injustice is complicity with the oppressor."](https://www.theguardian.com/news/2000/sep/14/guardianobituaries)
+
+Going beyond the hashtag, we have been discussing concrete actions we can take as a community and today we announce two efforts: first, to fully **[eliminate master/slave terminology](https://www.washingtonpost.com/opinions/2020/06/12/tech-industry-has-an-ugly-master-slave-problem/) from our code base** and second, to **recruit paid interns from underrepresented communities** to join our contributor base.
```

Comment:
```suggestion
Going beyond the hashtag, we have been discussing concrete actions we can take as a community and today we announce two efforts: first, to **[eliminate master/slave terminology](https://www.washingtonpost.com/opinions/2020/06/12/tech-industry-has-an-ugly-master-slave-problem/) from our code base** and second, to **recruit paid interns from underrepresented communities** to join our contributor base.
```
We can't do that. At least for COs we'll need to keep the term as an alias or we'd break backwards compatibility. We should explain that exception in the section below (e. g. "replace the term whereever we can without breaking third-party controller mappings or skins"?)

Reply:
Isn't it fully removed when we keep aliases? We also can't rewrite the history ...

Reply:
To me, fully removed means that you can `grep -HirnF master` the latest state of the repo and won't find anything.

Reply:
The sentence is about the master/slave terminology not master only.
fully is in contrast to half, because we have already removed slaves.
If you have a better alternative that can be read like that. I would be happy.

Reply:
Yeah, on second thought the word "fully" is misleading.  I'm going to remove it.